src/sentry/scim/endpoints/members.py

```diff
@@ -150,6 +152,21 @@ def post(self, request, organization):
             return Response(serializer.errors, status=400)
 
         result = serializer.validated_data
+
+        if ratelimits.for_organization_member_invite(
```

Comment:
I don't know if you want a rate limit here. The rate limit on invites was added to prevent using invites as a spamming vector. Accounts that will have access to SCIM are generally paying us, and don't really need to be rate limited as all the accounts they are inviting are known to them already.

Reply:
@markstory got it, wasn't sure about the context of those rate limits, that makes sense. I will close this for now then.